beacon-chain/blockchain/service.go

```diff
@@ -94,6 +94,49 @@ var ErrMissingClockSetter = errors.New("blockchain Service initialized without a
 type blobNotifierMap struct {
 	sync.RWMutex
 	notifiers map[[32]byte]chan uint64
+	seenIndex map[[32]byte]*[fieldparams.MaxBlobsPerBlock]bool
```

Comment:
Curious why the pointer and not just `map[[32]byte][fieldparams.MaxBlobsPerBlock]bool`?

Reply:
I was thinking reference is more straightforward than copy. like:
```
seenIndices := bn.seenIndex[root]
seenIndices[idx] = true
```

If it's not a pointer then I have to assign it back right?